webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -0,0 +1,124 @@
+  const filterType = errataByDate ? 'errata_by_date' : type;
+  const base = `/content_views/${cvId}/repositories/${repoType}/filters/${filterId}`;
+  const endings = {
+    rpm: '/package/details',
+    package_group: '/package-group/list',
+    erratum: '/errata/list',
+    errata_by_date: '/errata/date_type',
+    modulemd: '/module-stream/list',
+    docker: '/docker/details',
+  };
+
+  return base + endings[filterType];
+};
+
+const ContentViewFilters = ({ cvId }) => {
+  const response = useSelector(state => selectCVFilters(state, cvId), shallowEqual);
+  const status = useSelector(state => selectCVFiltersStatus(state, cvId), shallowEqual);
+  const error = useSelector(state => selectCVFiltersError(state, cvId), shallowEqual);
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+  const loading = status === STATUS.PENDING;
+
+  const columnHeaders = [
+    __('Name'),
+    __('Description'),
+    __('Updated'),
+    __('Content type'),
+    __('Inclusion type'),
+  ];
+
+  const buildRows = (results) => {
+    const newRows = [];
+    results.forEach((filter) => {
+      let errataByDate = false;
+      const {
+        id, name, type, description, updated_at: updatedAt, inclusion,
+      } = filter;
+      if (filter.type === 'erratum' && filter.rules[0].types) errataByDate = true;
+
+      const cells = [
+        { title: <a href={cvFilterUrl(cvId, id, type, errataByDate)}>{name}</a> },
+        truncate(description || ''),
+        { title: <LongDateTime date={updatedAt} showRelativeTimeTooltip /> },
+        { title: <ContentType type={type} errataByDate={errataByDate} /> },
+        {
+          title: (
+            <Label color={inclusion && 'blue'}>
+              {inclusion ? 'Include' : 'Exclude'}
+            </Label>),
+        },
+      ];
+
+      newRows.push({ cells });
+    });
+    return newRows;
+  };
+
+  useEffect(() => {
+    const { results, ...meta } = response;
+    setMetadata(meta);
+
+    if (!loading && results) {
+      const newRows = buildRows(results);
+      setRows(newRows);
+    }
+  }, [JSON.stringify(response)]);
+
+  const emptyContentTitle = __("You currently don't have any filters for this content view.");
+  const emptyContentBody = __("Add filters using the 'Add filter' button above."); // needs link
+  const emptySearchTitle = __('No matching filters found');
+  const emptySearchBody = __('Try changing your search settings.');
+
+  return (
+    <TableWrapper
+      {...{
+        rows,
+        metadata,
+        emptyContentTitle,
+        emptyContentBody,
+        emptySearchTitle,
+        emptySearchBody,
+        searchQuery,
+        updateSearchQuery,
+        error,
+        status,
+      }}
+      onSelect={onSelect(rows, setRows)}
+      cells={columnHeaders}
+      variant={TableVariant.compact}
+      autocompleteEndpoint="/content_view_filters/auto_complete_search"
```

Comment:
I'm still getting a 500 error on a search for `inclusion_type` (before I finish typing `= exclude`, for example.)  Is there an open issue for this?

Reply:
We are finding a lot of things with this autocomplete! [Here](https://projects.theforeman.org/issues/31523) is the filed issue.